### DISCUSSION TITLE
How does nicegui turn off the logic of sliding right back to the previous layer

### DISCUSSION THREAD

**Discussion by ponponon** [zauberzeug/nicegui#2565] [Q&A]
### Question

```python
from .header import add_head_html, add_header

def create() -> None:
    
    add_head_html()
    add_header()
```

When I import the add head html, add header, right swipe and a return function appears, but this conflicts with my right swipe to view the list

<img width="1434" alt="图片" src="https://github.com/zauberzeug/nicegui/assets/38725104/d8f678fe-78ec-4cea-b5b1-1046c949426c">

**Comment by rodja** [comment]
Sorry @ponponon, I do not quite understand your question. Could you please provide a minimal executable example and a clear description what is not working?
